common/test/acceptance/pages/studio/html_component_editor.py

```diff
@@ -238,6 +238,32 @@ def save_content(self):
         """
         click_css(self, '.action-save')
 
+    def open_image_modal(self):
+        """
+        Clicks and in insert image button
+        """
+        # self.q(css='div i[class="mce-ico mce-i-image"]').click()
+        click_css(self, 'div i[class="mce-ico mce-i-image"]')
+
+    def upload_image(self, file_name):
+        """
+        Upload the image via TinyMCE editor.
+        Upload image and add description and click save to upload image via TinyMCE editor.
+        """
+        file_input_css = "[type='file']"
```

Comment:
no reason to keep this in a variable as it is not being used elsewhere in the function